cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -1212,22 +1204,26 @@ def medium(self, key, value):
     _migration = self.get("_migration", {})
     item_mediums = _migration.get("item_medium", [])
     barcodes = []
+    _medium = None
     val_x = value.get("x")
 
     if val_x:
         barcodes = [barcode for barcode in force_list(val_x) if barcode]
 
-    _medium = mapping(
-        ITEMS_MEDIUMS,
-        clean_val("a", value, str).upper().replace("-", ""),
-        raise_exception=True,
-    )
+    val_a = clean_val("a", value, str)
+    if val_a:
+        _medium = mapping(
+            ITEMS_MEDIUMS,
+            val_a.upper().replace("-", ""),
+            raise_exception=True,
+        )
 
     for barcode in barcodes:
         current_item = {
             "barcode": barcode,
-            "medium": _medium,
         }
+        if _medium:
```

Comment:
I don't know if my comment makes sense, but `medium` is mandatory in the schema if I am not wrong

Reply:
we have the default medium on the import time. The rule is just trying to map any of the allowed mediums. Now I see that it would make sense to move the default here but I am worried to change it at this point. WDYT ?